CONTRIBUTORS

```diff
@@ -32,5 +32,6 @@ ChaiShushan <chaishushan@gmail.com>
 Christoph Hack <christoph@tux21b.org>
 Markus Sonderegger <marraison@gmail.com>
 Matt Proud <matt.proud@gmail.com>
+Mike Wiacek <mjwiacek@google.com>
```

Comment:
OK, it still shows up in github's PR page, but I'm relatively new to git and github, so I might be misinterpreting it.